<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room_Dialog</class>
 <widget class="QDialog" name="room_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>room_Dialog</string>
  </property>
  <widget class="QWidget" name="wdg_enemy" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QPushButton" name="pb_enemy_headshot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>敌人头像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_enemy_skill">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>敌人技能</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="SA_enemy_cardlist">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>120</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>600</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="wdg_enemy_cardshow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>598</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_enemyid">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>玩家</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wdg_player" native="true">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>120</width>
     <height>720</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pb_player_headshot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>玩家头像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_player_skill">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>玩家技能</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="SA_player_cardlist">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>120</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>600</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="wdg_player_cardshow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>598</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_playerid">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>玩家</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wdg_fight" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>720</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>720</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QLabel" name="lb_roomid">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>roomid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_exitroom">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>投降</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="SA_enemy_cardlist_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>720</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="wdg_enemy_fight_cardshow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>718</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="SA_player_cardlist_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>120</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>720</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="wdg_player_fight_cardshow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>718</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
